litmus-portal/frontend/src/views/AnalyticsDashboard/MonitoringDashboardPage/ChaosTable/TableData.tsx

```diff
@@ -0,0 +1,65 @@
+import { Typography } from '@material-ui/core';
+import React from 'react';
+import CheckBox from '../../../../components/CheckBox';
+import { ChaosEventDetails } from '../../../../models/dashboardsData';
+import useStyles, { StyledTableCell } from './styles';
+
+interface TableDataProps {
+  data: ChaosEventDetails;
+  itemSelectionStatus: boolean;
+  labelIdentifier: string;
+}
+
+const TableData: React.FC<TableDataProps> = ({
+  data,
+  itemSelectionStatus,
+  labelIdentifier,
+}) => {
+  const classes = useStyles();
+
+  return (
+    <>
```

Comment:
Avoid <>

Reply:
Its required for the table rows, cannot be changed to div as that reduces colspan to 1 for that row. you can take a look at other tables for rference.

Reply:
cc: @arkajyotiMukherjee

Reply:
this is fine it seems...as there are multiple tablecells being exported @whitetiger1399